modules/ossm-extensions-container-format.adoc

```diff
@@ -0,0 +1,57 @@
+////
+This module included in the following assemblies:
+*service_mesh_/v2x/ossm-extensions.adoc
+////
+:_content-type: REFERENCE
+[id="ossm-extensions-format_{context}"]
+= WASM container format
```

Comment:
note that this format is only for `ServiceMeshExtension` (ie the legacy API). The new WasmPlugin API uses this format: https://github.com/istio-ecosystem/wasm-extensions/blob/master/doc/how-to-build-oci-images.md

the big change is that the `manifest.yaml` does not exist in that format, and the .wasm file always needs to be called `plugin.wasm`

Reply:
"the .wasm file always needs to be called plugin.wasm"

Does that mean that users are restricted to only being able to create a single plugin?

Reply:
within a single container: yes. but that has been the case with ServiceMeshExtension as well